src/Viewer/services/utils.js

```diff
@@ -43,6 +43,25 @@ const modifyPage = (action, currentPage, requestedPage, pages) => {
     return [linePos, newPage];
 };
 
+/**
+ * Returns an absolute URL of a given path relative to the window.location,
+ * if the given path is a relative reference.
```

Comment:
```suggestion
 * Gets an absolute URL composed of a given path relative to the
 * window.location, if the given path is a relative reference; otherwise
 * the given path is returned verbatim.
```